### DISCUSSION TITLE
ipmitool sensor column headers/value interpretation

### DISCUSSION THREAD

**Discussion by nank22** [ipmitool/ipmitool#348] [Q&A]
I'm not able to find any documentation that explains what the entries of the different columns are for the sensor command. In one of the bug reports on sourceforge.net, someone provided the following info:
for analog sensors, the header is: 
   Name | AnalogValue | Unit | [ok|nr|cr|nc|ns] | Inr value | Icr value | Incr value | uncr value | uc value | unr value
where: nr: non-recoverable; cr: critical; nc: non-critical; ns: not specified
but there was no explanation/interpretation provided for Inr/Icr/Incr/uncr/uc/unr
Can anyone help?

**Comment by doublechiang** [comment]
l and u are representing low and upper. 
So uc is upper critical.
